### PR TITLE
Update docs/README.md

### PR DESCRIPTION
**Pull request by ydshieh**
# What does this PR do?

Add information about how to perform doc testing **for a specific python module**. Potentially making the process less scaring (a lot of file changes) for the users. 

Also make it more clear/transparent that `file_utils.py` should be (almost) always included.

### CODE REVIEW

**Review by sgugger — COMMENTED** [review]
That's not accurate: it works here because the docstring with example are all defined in `file_utils.py` but for another file, this wouldn't work.

**Comment by ydshieh** [comment]
Do you mean: there are some python scripts or doc files, for which if we want to run doctest, it will require changes in other files other than 

- the target files themselves
-  `file_utils.py`.

**Review by sgugger — APPROVED** [review]
Ah sorry, I misread the command! This works indeed but could we maybe wait for the refactor of `file_utils` to merge this (and fix the paths)? I'll probably forget to adapt this otherwise.

**Comment by ydshieh** [comment]
> Ah sorry, I misread the command! This works indeed but could we maybe wait for the refactor of `file_utils` to merge this (and fix the paths)? I'll probably forget to adapt this otherwise.

Oh, yes for sure!

File: docs/README.md
Comment (LysandreJik) [inline docs/README.md:352]:
```suggestion
If you work on a specific python module, say `modeling_wav2vec2.py`, you can run the command as follows (to avoid the unnecessary temporary changes in irrelevant files):
```
`script` implies that it should be run, which is something that is not encouraged/will generally not work.

**Review by LysandreJik — APPROVED** [review]
LGTM

**Review by patrickvonplaten — APPROVED** [review]

**Comment by ydshieh** [comment]
Will merge this PR once Sylvain's PR about file_utils is merged.

File: docs/README.md
Comment (sgugger) [inline docs/README.md:357]:
```suggestion
(`utils/doc.py` should be always included)
```

**Comment by ydshieh** [comment]
@sgugger Should I wait a bit before merge? Checks are not run, maybe it is because GitHub is down?

**Pull request merged**
